Jenkinsfile

```diff
@@ -1,34 +1 @@
    try {
        block()
    }
    catch (Throwable t) {
        slackSend message: "${env.JOB_NAME}: ${message} failure ${env.BUILD_URL}", channel: '#ci-channel', color: 'danger'
        throw t
    }
    finally {
        if (tearDown) {
            tearDown()
        }
    }
}

node {
    stage("Checkout") {
        checkout scm
    }

    stage("Build API image") {
        tryStep "build", {
            docker.withRegistry("${DOCKER_REGISTRY_HOST}",'docker_registry_auth') {
            def image = docker.build("datapunt/dataservices/publish-static:${env.BUILD_NUMBER}", ".")
            image.push()
            }
        }
    }
}


node {
    stage("Push ACC image") {
        tryStep "image tagging", {
            docker.withRegistry("${DOCKER_REGISTRY_HOST}",'docker_registry_auth') {
            def image = docker.image("datapunt/dataservices/publish-static:${env.BUILD_NUMBER}")
            image.pull()
            image.push("acceptance")
            }
        }
    }

    stage("Deploy to ACC") {
        tryStep "deployment", {
            build job: 'Subtask_Openstack_Playbook',
                    parameters: [
                            [$class: 'StringParameterValue', name: 'INVENTORY', value: 'acceptance'],
                            [$class: 'StringParameterValue', name: 'PLAYBOOK', value: 'publish-dataservices-static-schemas.yml'],
                    ]
       }
   }

    stage("Import schemas into DSO-API ACC DB") {
        tryStep "import", {
            build job: 'ams_schema_importer',
                parameters: [
                    [$class: 'StringParameterValue', name: 'environment', value: 'acceptance']
                ]
        }
    }
}

stage('Waiting for approval') {
    slackSend channel: '#ci-channel', color: 'warning', message: 'Amsterdam Schema is waiting for Production Release - Please confirm.'
    input "Deploy to Production?"
}

node {
    stage("Push PROD image") {
        tryStep "image tagging", {
            docker.withRegistry("${DOCKER_REGISTRY_HOST}",'docker_registry_auth') {
            def image = docker.image("datapunt/dataservices/publish-static:${env.BUILD_NUMBER}")
            image.pull()
            image.push("production")
            }
        }
    }

    stage("Deploy to PROD") {
        tryStep "deployment", {
            build job: 'Subtask_Openstack_Playbook',
                    parameters: [
                            [$class: 'StringParameterValue', name: 'INVENTORY', value: 'production'],
                            [$class: 'StringParameterValue', name: 'PLAYBOOK', value: 'publish-dataservices-static-schemas.yml'],
                    ]
        }
    }

    stage("Import schemas into DSO-API PROD DB") {
        tryStep "import", {
            build job: 'ams_schema_importer',
                parameters: [
                    [$class: 'StringParameterValue', name: 'environment', value: 'production']
                ]
        }
    }
}
```

Comment:
`^M` not cool.

Reply:
That's bizarre. I did notice the pre-commit _Trim Trailing Whitespace_ running twice in succession on this file, but can't imagine it doing something like that. Anyway should be fixed in my next push.